### DISCUSSION TITLE
Showing pics of multiple albums in collection

### DISCUSSION THREAD

**Discussion by imsodin** [deckerst/aves#1012] [Q&A] [answered]
I searched quite a bit for this in the app or here on github, but couldn't find anything: Can I display/search for pics from multiple albums?

I can select multiple tags one by one on the search page, though that ANDs. I can't do the same for albums, possibly because ANDing makes no sense there.

Long pressing doesn't give me any options towards that (though I found the "filter out" option there, pretty well hidden).

On the album page I can select multiple albums, and even display them on a map or on the slideshow, but not the collection/as thumbnails.

What I want to do: I have a dir/album with my own camera images, and one with ones synced from my partners phone. We are traveling together and I'd like to view and tag them from a single timeline. I can sort of achieve that by filtering out albums that currently introduce new pics to the global timeline, but that's cumbersome and backwards. By the way excluding multiple albums works :)

And side-questions: The only docs are the wiki, right? I was thinning whether done of my confusions and discoveries might be worth contributing as docs, but don't see how/where.

**Comment by deckerst** [comment]
All your observations are correct.

For now, I don't have a convenient solution for you.

For the future, allowing a multi-album selection to be displayed in the Collection page (just like it's done on the map, as you pointed out) would be an elegant approach. Thanks for the idea.

**Comment by kmcbest** [comment]
From what I understand, your photos and your partner's are very close in creation time, why aren't they shown up next to each other in the collection page?

**Reply by imsodin** [reply]
oh they are, but others are interjected. we are lucky to be traveling for quite long time, so lots of opportunities for other pics to get onto the phone (chats basically). Anyway sounds like multi-album selection is the generally useful concept here - happy that I could being up the idea :)

**Comment by deckerst** [comment, marked as answer]
> allowing a multi-album selection to be displayed in the Collection page

Done for the next release. The combined filter is usable like others: shortcut, home, widget, hide, filter out, etc.
